alea/template_source.py

```diff
@@ -125,8 +125,8 @@ def build_histogram(self):
         # Shouldn't be in HistogramSource... anyway
         self._n_events_histogram = h.similar_blank_histogram()
 
-        h *= self.config.get('histogram_multiplier', 1)
-        logging.debug(f"Multiplying histogram with histogram_multiplier {self.config.get('histogram_multiplier', 1)}. Histogram is now normalised to {h.n}.")
+        h *= self.config.get('histogram_scale_factor', 1)
+        logging.debug(f"Multiplying histogram with histogram_scale_factor {self.config.get('histogram_scale_factor', 1)}. Histogram is now normalised to {h.n}.")
```

Comment:
This line is too long. 

```
logging.debug(
    f"Multiplying histogram with histogram_scale_factor {self.config.get('histogram_scale_factor', 1)}."
    f"Histogram is now normalised to {h.n}.")
```

or something else.

Reply:
I think this anyway goes (all the debugging) so I wouldn't bother now cleaning up legacy code.